Files/Files.csproj

```diff
@@ -266,6 +266,8 @@
     <Content Include="Assets\Files UWP Icon.png" />
     <Content Include="Assets\FilesHome.png" />
     <Content Include="Assets\FolderIcon.svg" />
+    <Content Include="Assets\GridViewLarge.png" />
+    <Content Include="Assets\GridViewSmall.png" />
```

Comment:
Are the Segoe UI assets you can use instead of these?

Reply:
Yes, I've used the FontIcon fonts @tsvietOK suggested, it looks much better in the dark theme.

I'll commit the changes as soon as I've fixed the folder bug.